packages/server-wallet/src/models/channel.ts

```diff
@@ -144,10 +144,14 @@ export class Channel extends Model implements RequiredColumns {
       .findOne({channelId}));
   }
 
-  static allLedgerChannels(txOrKnex: TransactionOrKnex): Promise<Channel[]> {
+  static getLedgerChannels(
+    assetHolderAddress: string,
+    participants: Participant[],
+    txOrKnex: TransactionOrKnex
+  ): Promise<Channel[]> {
     return Channel.query(txOrKnex)
       .select()
-      .whereNotNull('assetHolderAddress');
```

Comment:
The before looks incorrect here, but maybe I am not familiar with the wallet schema?

Reply:
`assetHolderAddress` is being set to `NULL` for channels which are not ledger channels (cc @geoknee)